xline/src/storage/kv_store.rs

```diff
@@ -934,7 +1059,10 @@ mod test {
     #[abort_on_panic]
     async fn test_recover() -> Result<(), ExecuteError> {
```

Comment:
We'd better to test recovery failure case, for example, the compation revision is not available.

Reply:
FYI: #375